Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
We had issues in the past with it on Vox Pupuli, not only on the gems but also the modules. It worked fine this time for me, but my Archlinux is also pretty bleeding edge. If you want I can keep it, but something in me tells me to not track it (worst argument ever).